src/sqlfluff/rules/L059.py

```diff
@@ -13,16 +13,18 @@
 class Rule_L059(BaseRule):
     """Unnecessary quoted identifier.
 
-    | **Anti-pattern**
-    | In this example, a valid unquoted identifier,
-    | that is also not a reserved keyword, is needlessly quoted.
+    **Anti-pattern**
+
+    In this example, a valid unquoted identifier,
+    that is also not a reserved keyword, is needlessly quoted.
```

Comment:
```suggestion
    In this example, an identifier is unnecessarily quoted. (The quotes are unnecessary because
    it is not a reserved keyword.)
```

Reply:
```suggestion
    In this example, an identifier is unnecessarily quoted. The quotes are unnecessary because
    it is not a reserved keyword.
```